### PR TITLE
[Routing] marked configurators traits as internal

### PR DESCRIPTION
**Pull request by HeahDude**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no 
| Deprecations? | no 
| Tickets       | Fix https://github.com/symfony/symfony/pull/30501#discussion_r376806342 
| License       | MIT
| Doc PR        | not needed

### CODE REVIEW

File: src/Symfony/Component/Routing/Loader/Configurator/Traits/AddTrait.php [hunk 1]
```diff
@@ -16,6 +16,8 @@
 use Symfony\Component\Routing\RouteCollection;
 
 /**
+ * @internal
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Routing/Loader/Configurator/Traits/AddTrait.php]: should be removed

File: src/Symfony/Component/Routing/Loader/Configurator/Traits/RouteTrait.php [hunk 2]
```diff
@@ -14,6 +14,9 @@
 use Symfony\Component\Routing\Route;
 use Symfony\Component\Routing\RouteCollection;
 
+/**
+ * @internal
```
Comment (nicolas-grekas) [inline src/Symfony/Component/Routing/Loader/Configurator/Traits/RouteTrait.php]: should be removed

**Comment by nicolas-grekas** [comment]
Only traits new to 5.1 should be made internal.

**Comment by HeahDude** [comment]
Alright, fixed.

**Review by nicolas-grekas — APPROVED** [review]

**Comment by nicolas-grekas** [comment]
Thank you @HeahDude.

**Pull request merged**